src/Microsoft.ApplicationInsights.AspNet/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -38,10 +40,10 @@ public static IApplicationBuilder UseApplicationInsightsExceptionTelemetry(this
             return app.UseMiddleware<ExceptionTrackingMiddleware>();
         }
 
-        public static void AddApplicationInsightsTelemetry(this IServiceCollection services, IConfiguration config, bool reuseActiveConfig = true)
+        public static void AddApplicationInsightsTelemetry(this IServiceCollection services, IConfiguration config, bool disableDefaultAdaptiveSampling = false)
```

Comment:
Chatted with David, let's discuss this some more tomorrow before we merge to master.
1. Consider wrapping this flag in a config structure to allow future changes
2. Customers who onboarded to previous version with reuseActiveConfig will not be able to notice anything changed and the fact that now this flag has completely different meaning.